extensions/email/src/main/java/org/mule/extension/email/internal/retriever/imap/IMAPSProvider.java

```diff
@@ -28,21 +32,27 @@
  * @since 4.0
  */
 @Alias("imaps")
+@DisplayName("IMAPS Connection")
 public class IMAPSProvider extends AbstractRetrieverProvider<RetrieverConnection> implements Initialisable
 {
 
     /**
-     * The port number of the mail server.
+     * The port number of the mail server. '993' by default.
      */
     @Parameter
     @Optional(defaultValue = IMAPS_PORT)
+    @Summary("The port number of the mail server")
+    @Placement(tab = "General", group = "Connection", order = 2)
     private String port;
 
     /**
      * A factory for TLS contexts. A TLS context is configured with a key store and a trust store.
      * Allows to create a TLS secured connections.
      */
     @Parameter
+    @Summary("TLS Context for the secure connection of IMAPS protocol")
+    @Placement(tab = "General", group = "Connection", order = 5)
+    @DisplayName("TLS Context Factory")
```

Comment:
TLS Context factory is not a meaningful name for this. From the user point of view this is TLS configuration, he doesn't know or care that we handle it through a factory.